.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -11,7 +11,7 @@ If your PR only contains changes to documentation, you may skip the template bel
 _Please make sure to review and check all of these items:_
 
 - [ ] Have you added new tests to prevent regressions?
-- [ ] Does `npm run test` or `npm run test-DIALECT` pass with this change (including linting)?
+- [ ] Does `npm run test` / `yarn test` or `npm run test-DIALECT` / `yarn test-DIALECT` pass with this change (including linting)?
```

Comment:
`npm run test` uses `yarn` internally so the user *has* to have it. We could drop `npm run test` here and only list the yarn version.

I would propose:
- All commands aimed at maintainers only list yarn
- Commands aimed at the users of the library list both yarn & npm, separated by a slash

Reply:
Oh right, that's a good one. I will update the PR accordingly